lib/cc/engine/analyzers/violation.rb

```diff
@@ -57,7 +61,15 @@ def name
         end
 
         def calculate_points
-          base_points * issue.mass
+          if issue.mass > threshold
```

Comment:
I think this should be `>=`? I think something that exactly hit the mass threshold is still an issue.